src/test/java/seedu/intrack/model/internship/TaskTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.intrack.model.internship;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.intrack.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Task(null, null));
+    }
```

Comment:
Would it be better to test the two null inputs separately? Following the "no more than one invalid input in a test case" heuristic.

Reply:
Thank you for the suggestion, I will update the tests accordingly.